src/panels/lovelace/cards/hui-weather-forecast-card.ts

```diff
@@ -189,6 +196,12 @@ class HuiWeatherForecastCard extends LitElement implements LovelaceCard {
       dayNight = dayDiff > DAY_IN_MILLISECONDS;
     }
 
+    const twice_daily =
+      this._config?.forecastType === "twice_daily" ? true : undefined;
```

Comment:
Not just when the user set it explicitly, but always when `getForecast` returns a `twice_daily` forecast.